hw/src/trezor/gen/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod types;
+pub mod messages;
```

Comment:
is it possible to generate those files at compile time rather than committing 25k loc? :)

Reply:
doubling request, this code should be committed only if compile-time generation is absolutely impossible

Reply:
While not impossible, there is no existing way to do it. Down the line we could write a macro to do some compile-time generation, but it's pretty hard to do without requiring the protobuf C-library from google. An acceptable intermediate solution that I've now pushed is to put all the generated code in the `trezor-sys` crate and import what we need from there.